ui/src/app/icons/scroll/component.tsx

```diff
@@ -0,0 +1,50 @@
+import {Component, Input} from '@angular/core';
+import {ReactWrapperBase} from 'app/utils';
+
+import {ScrollIconReact} from 'app/icons/scroll-icon/component';
+import * as React from 'react';
+import * as Interactive from 'react-interactive';
+
+const FillColorId = '#2691D0';
+const HoverFillColorId = '#72B9E2';
+const LightFillColorId = '#5FAEE0';
+const LightHoverFillColorId = '#1892E0';
+
+const wrapper = ScrollIconReact;
+export const ScrollReact: React.FunctionComponent<{icon: string, shade: string, style: object}> =
```

Comment:
Let's stop putting 'React' in the name, since that seems somewhat redundant. Going forward I think we should follow the pattern of naming react components like `FooBar`, and the corresponding angular wrapper like `FooBarComponent`. Thoughts?

Reply:
I agree, have updated the file accordingly.